packages/plans-grid-next/src/components/features-grid/client-logo-list/index.tsx

```diff
@@ -6,21 +6,18 @@ const ClientLogoList = ( props: React.HTMLAttributes< HTMLDivElement > ) => {
 	const { className, ...otherProps } = props;
 
 	const elementProps: React.HTMLAttributes< HTMLDivElement > = {
-		className: clsx( 'client-logo-list', className ),
+		className: clsx( 'plans-grid-next-features-grid-client-logo-list', className ),
 		...otherProps,
-		role: 'presentation',
-		'aria-hidden': 'true',
 	};
 
 	return (
 		<div { ...elementProps }>
 			{ CLIENT_LIST.map( ( { slug, name, Logo } ) => (
 				<div
 					key={ slug }
-					className={ `client-logo-list__item client-logo-list__item--name-${ slug }` }
-					title={ name }
```

Comment:
I'm assuming this `title` was for the hover effect. Moving this to a `title` tag inside the SVG accomplishes the same thing.